.s2i/bin/assemble

```diff
@@ -17,6 +17,11 @@ cp -vr /tmp/src/ /opt/app-root/wildcard-router/
 mkdir -p /opt/app-root/wildcard-router/logs
 chmod g+w /opt/app-root/wildcard-router/logs
 
+# FIXME: otherwise nginx will get:
+# nginx: [emerg] mkdir() "/opt/app-root/wildcard-router/client_body_temp" failed (13: Permission denied)
+mkdir -p /opt/app-root/wildcard-router/client_body_temp
+chmod g+w /opt/app-root/wildcard-router/client_body_temp
```

Comment:
Maybe we should not change the prefix when starting nginx and it would be a bit easier.